patches/tModLoader/Terraria/ModLoader/Core/ModOrganizer.cs

```diff
@@ -319,5 +325,77 @@ internal static class ModOrganizer
 				return new HashSet<string>();
 			}
 		}
+
+		private static readonly Regex PublishFolderMetadata = new Regex(@"([0-9][0-9][0-9][0-9]*[.][0-9]*)");
```

Comment:
Is the regex meant to match on the following?
`2022222.`
`202.`
`202.222222222222222`
2022.2222222`
`2022000000.222222222`